massa-graph/src/block_graph.rs

```diff
@@ -560,7 +604,21 @@ impl BlockGraph {
             Map::with_capacity_and_hasher(required_active_blocks.len(), BuildMap::default());
         for b_id in required_active_blocks {
             if let Some(BlockStatus::Active(a_block)) = self.block_statuses.get(&b_id) {
-                active_blocks.insert(b_id, ExportActiveBlock::from(&**a_block));
+                let block = self.storage.retrieve_block(&b_id).unwrap();
```

Comment:
This function is returning an error maybe consider map the `None` to an Error with `ok_or` instead of unwrap.

Reply:
Yes, thanks. I will address this and others in a separate PR that will close https://github.com/massalabs/massa/issues/2381